packages/react/package.json

```diff
@@ -48,6 +51,8 @@
     "@gadgetinc/api-client-core": "^0.15.24",
     "@graphql-typed-document-node/core": "^3.2.0",
     "@hookform/resolvers": "^3.3.1",
+    "@types/tmp": "^0.2.6",
+    "copyfiles": "^2.4.1",
```

Comment:
Out of curiousity, what goes into the decision of making these `dependencies` vs `devDependencies`. It that they are not referenced by prodution facing files

Reply:
They should be devDependencies, good catch.